README.rst

```diff
@@ -144,7 +144,7 @@ To set the layout on boot, add it to your ``plan9.ini(8)``::
           ...
         % unmount /n/9fat
 
-Ensure that the file ends in a newline. You will also need to rebuild your kernel, following the instructions under `FQA 5`_. 
+Ensure that the file ends in a newline. You will also need to rebuild your kernel, following the instructions under `FQA 5`_.
```

Comment:
Meant to make a separate commit?

Reply:
not really, I just have nvim configured to automatically delete trailing spaces when saving